connectors/es/client.py

```diff
@@ -164,23 +164,16 @@ async def wait(self):
         await self.close()
         return False
 
-    async def check_exists(self, indices=None, pipelines=None):
+    async def ensure_exists(self, indices=None):
         if indices is None:
             indices = []
-        if pipelines is None:
```

Comment:
do the indices templates define the pipelines? is there still a need to check them?